client/apps/account-settings/components/label-payment-method/index.js

```diff
@@ -32,18 +32,18 @@ const PaymentMethod = ( { translate, selected, type, digits, name, expiry, onSel
 	};
 
 	return (
-		<CompactCard className="shipping__card" onClick={ onSelect }>
+		<CompactCard className="label-payment-method" onClick={ onSelect }>
```

Comment:
I'm OK with that, there's nothing specific about labels in that component. Renamed in https://github.com/Automattic/woocommerce-services/pull/1051/commits/3a9ba3e58d8b5f9b901fcbce8ffb63df960c7e70